client/src/Components/App/index.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
remove code from this file 
@asem1789 will make an issue and fix this bug in split PR